keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+import argparse
+import pycparser
+
+def beautify_src_file(filename):
+    src = {
+        "before": [],
+        "keys": [],
+        "after": []
+    }
+
+    with open(filename) as f:
+        current_section = "before"
+        for line in f.read().split("\n"):
+            if current_section == "before":
+                if line == "const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {":
+                    current_section = "keys"
+                    src[current_section].append("const int keymaps[]={")
```

Comment:
The parser doesn't parse the original KEYMAP_START, so I temporarily replaced it. It will be put back in correctly later.

I'll be working on the padding first, and then perhaps you can help with the styling and clarity. Thanks again!